src/systems/filecoin_blockchain/storage_power_consensus/storage_power_actor.id

```diff
@@ -1,26 +1,41 @@
 import filcrypto "github.com/filecoin-project/specs/libraries/filcrypto"
 import addr "github.com/filecoin-project/specs/systems/filecoin_vm/actor/address"
+import libp2p "github.com/filecoin-project/specs/libraries/libp2p"
 
 type BytesAmount UVarint
 
 type StoragePowerActor struct {
+    Miners [addr.Address]  // TODO this should have a different type to account for power
+
     // call by StorageMiningSubsytem on miner creation
-    RegisterMiner(ownerAddr addr.Address, workerPubKey filcrypto.PubKey) addr.Address
+    CreateStorageMiner(
+        ownerAddr     addr.Address
+        workerPubKey  filcrypto.PubKey
+        sectorSize    util.UInt
+        peerId        libp2p.PeerID
```

Comment:
I understand this was here before. This is not a good registry of libp2p keys -- they may change frequently, and libp2p keys need to be associated with more than just storage miners (retrieval miners, clients, bootstrap nodes). I'm OK adding this to reflect status quo, but expect this to be moved elsewhere before mainnet.

Reply:
ok, will add a TODO comment to it.